demos/k8s-demo/README.md

```diff
@@ -88,12 +104,12 @@ configmap "quick-start-application-secretless-config" created
 to the database.**
 
 In particular, as the application developer you do not know any of the secrets
-in **security_admin_secrets.sh**.
+in **security_admin_vars.sh**.
```

Comment:
I don't quite think I agree with either this comment nor the one about `env.sh` since "vars" and "env" are somewhat universal but at this point we're needlessly bikeshedding this PR so I'll make the change.